helix-term/src/commands.rs

```diff
@@ -2267,6 +2268,17 @@ fn file_picker(cx: &mut Context) {
     cx.push_layer(Box::new(overlayed(picker)));
 }
 
+fn file_picker_in_current_view_directory(cx: &mut Context) {
+    let (_, doc) = current_ref!(cx.editor);
+
+    let cwd = doc
+        .path()
+        .and_then(|path| path.parent().map(|path| path.to_path_buf()))
+        .unwrap_or_else(|| std::env::current_dir().unwrap_or_else(|_| PathBuf::from("./")));
```

Comment:
For this command, instead of falling back to the CWD, I think we should `cx.editor.set_error` a message that says that the current buffer doesn't have a directory and return early